src/Microsoft.ML.CpuMath/CpuMathUtils.netcoreapp.cs

```diff
@@ -24,757 +24,757 @@ public static partial class CpuMathUtils
-                    Contracts.Assert(crun <= src.Size);
-                    SseIntrinsics.MatMulTranA(add, mat, src, dst, dst.Size, crun);
+                    Contracts.Assert(stride <= source.Size);
+                    SseIntrinsics.MatMulTranA(add, mat, source, destination, destination.Size, stride);
                 }
             }
             else
             {
-                if (!tran)
+                if (!transpose)
                 {
-                    Contracts.Assert(crun <= dst.Size);
-                    for (int i = 0; i < crun; i++)
+                    Contracts.Assert(stride <= destination.Size);
+                    for (int i = 0; i < stride; i++)
                     {
                         float dotProduct = 0;
-                        for (int j = 0; j < src.Size; j++)
+                        for (int j = 0; j < source.Size; j++)
                         {
-                            dotProduct += mat[i * src.Size + j] * src[j];
+                            dotProduct += mat[i * source.Size + j] * source[j];
                         }
 
                         if (add)
                         {
-                            dst[i] += dotProduct;
+                            destination[i] += dotProduct;
                         }
                         else
                         {
-                            dst[i] = dotProduct;
+                            destination[i] = dotProduct;
                         }
                     }
                 }
                 else
                 {
-                    Contracts.Assert(crun <= src.Size);
-                    for (int i = 0; i < dst.Size; i++)
+                    Contracts.Assert(stride <= source.Size);
+                    for (int i = 0; i < destination.Size; i++)
                     {
                         float dotProduct = 0;
-                        for (int j = 0; j < crun; j++)
+                        for (int j = 0; j < stride; j++)
                         {
-                            dotProduct += mat[j * src.Size + i] * src[j];
+                            dotProduct += mat[j * source.Size + i] * source[j];
                         }
 
                         if (add)
                         {
-                            dst[i] += dotProduct;
+                            destination[i] += dotProduct;
                         }
                         else
                         {
-                            dst[i] = dotProduct;
+                            destination[i] = dotProduct;
                         }
                     }
                 }
             }
         }
 
-        public static void MatTimesSrc(bool tran, bool add, AlignedArray mat, int[] rgposSrc, AlignedArray srcValues,
-            int posMin, int iposMin, int iposLim, AlignedArray dst, int crun)
+        public static void MatrixTimesSource(bool transpose, bool add, AlignedArray mat, int[] rgpossource, AlignedArray sourceValues,
```

Comment:
We shouldn't need the `rg` prefix, we can just use `posSource`. 
This `mat` (and most of the others in the file) also stands for `matrix`